src/interfaces/rootgame.ts

```diff
@@ -302,5 +302,5 @@ export interface RootGame {
   players: Partial<Record<RootFaction, string>>;    // { [factionkey]: player }
   turns: RootTurn[];                                // all of the game turns in order
   winner: RootFaction[];                            // the winner(s) of the game
-
+  stats: any
```

Comment:
Can you make this a RootStats interface? If we're going to track stats, I'd like to have them typed for when they get consumed.

I'm also hesitant to include this to the parser. Maybe this could be a secondary export, `calculateGameStats` that isn't tied into the `RootGame` interface?

Reply:
I'll just make it a calculating function that doesn't tie into the `RootGame` interface at all. That's a good call.

Reply:
How about as it is now? Or would you like the substat objects to also be typed?

Reply:
For now, I think we should remove the `stats` prop until we have more to work with. It sounds like there will be more!

Reply:
I agree. This is more maintainable. :)